consensus/src/event_processor.rs

```diff
@@ -840,6 +845,11 @@ impl<T: Payload> EventProcessor<T> {
                 .certified_block()
                 .round(),
         );
+        // @REVIEW: The strategy taken here (where you bootstrap by replaying old events) is very
+        // different than the one taken by EpochManager::start_event_processor (which does a fair
+        // bit of manual reconstruction). I wonder if start_event_processor would benefit from a similar
```

Comment:
I tried the same strategy (replay all blocks), the only manual reconstruction is the virtual genesis because we never persist it (plus we do some cleanup)